examples/simple_regional_with_gateway_api/main.tf

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  cluster_type = "simple-regional"
```

Comment:
Please update the cluster type accordingly

Reply:
simple-regional -> simple-regional-with-gatewayapi

Reply:
Done. Can you shortly explain to me why this is necessary? Or is it just to keep everything aligned with some kind of naming convention?

Reply:
Your understanding is right. In this way, it's easier to identify the testing clusters by its name.

Reply:
But I guess this now leads to an issue in the integration check as the cluster_name attribute is wrongly set?
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1510/files#diff-0e66d26ba66e1418a2d27bff7f636be5977955a71c6222b5e2f9f6f586d21180R15-R25

https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1510/files#diff-ab53d5a46fde1a871e35ca555d6f5dc58fcac8fa7cddd9c1fa188c125ce672d8R17

Reply:
It's the length of the cluster exceeding the limit.